### ISSUE TITLE
Icon Request: Cryptomator

### ISSUE THREAD

**Issue by toplac** [numixproject/numix-core#3569]
Homepage: https://cryptomator.org/de/

Desktop File: 

```
[Desktop Entry]
Name=Cryptomator
Comment=Cloud Storage Encryption Utility
Exec=/usr/bin/cryptomator
Icon=/usr/share/java/cryptomator/Cryptomator.png
Terminal=false
Type=Application
Categories=Utility
MimeType=application/x-vnd.cryptomator-vault-metadata
```

Logo: ![lpgp](https://cryptomator.org/img/stage/logo.png)

**Comment by Foggalong** [comment]
As a note about markdown, you use this formatting to make sure code displays correctly:

\```
code goes here
\```

You can then also use \!\[](image.url) to display images inline.

**Comment by bilelmoussaoui** [comment]
@toplac, we will need the desktop file name too as the icon is hardcoded

**Comment by toplac** [comment]
@Foggalong Thx for the hint

@bil-elmoussaoui 
/usr/share/applications/cryptomator.desktop

**Comment by palob** [comment]
Fixed, see #3957.
